modules/tensor_mechanics/include/materials/ADHillPlasticityStressUpdate.h

```diff
@@ -85,7 +85,8 @@ class ADHillPlasticityStressUpdate : public ADAnisotropicReturnPlasticityStressU
                                ADReal & sy_gamma);
 
   /// Hill constants for orthotropic creep
-  std::vector<Real> _hill_constants;
+  std::vector<Real> _hill_constants_input;
```

Comment:
Can we do the same orientation transformation for Hill's creep?

Reply:
Done!